SPECS-EXTENDED/kernel-ipe/config

```diff
@@ -545,6 +545,7 @@ CONFIG_CPU_SRSO=y
 CONFIG_MITIGATION_RFDS=y
 CONFIG_MITIGATION_SPECTRE_BHI=y
 CONFIG_MITIGATION_ITS=y
+CONFIG_MITIGATION_TSA=y
```

Comment:
result of https://github.com/microsoft/CBL-Mariner-Linux-Kernel/commit/90293047df18caf7ed0fe566d89af0720f2dc98a